docs/sources/developers/plugins/backend/_index.md

```diff
@@ -0,0 +1,62 @@
++++
+title = "Backend plugins"
+keywords = ["grafana", "plugins", "backend", "plugin", "backend-plugins", "documentation"]
+type = "docs"
+aliases = ["/docs/grafana/latest/plugins/developing/backend-plugins-guide/"]
++++
+
+# Backend plugins
+
+Grafana added support for plugins in Grafana 3.0 and this enabled the Grafana community to create panel plugins and data source plugins. It was wildly successful and has made Grafana much more useful as you can integrate it with anything and do any type of custom visualization that you want.
+
+However, one limitation with these plugins are that they execute on the client-side (in the browser) whick makes it hard to support  certain use cases/features, e.g. enable Grafana Alerting for data sources. Grafana v7.0 adds official support for backend plugins which removes this limitation. At the same time it gives plugin developers the possibility to extend Grafana in new interesting ways, with code running in the backend (server side).
+
+We use the term *backend plugin* to denote that a plugin has a backend component. Still, normally a backend plugin requires frontend components as well. This is for example true for backend data source plugins which normally needs configuration and query editor components implemented for the frontend.
+
+Currently, data source plugins can be extended with a backend component. In the future we plan to support additional types and possibly new kinds of plugins, e.g. [notifiers for Grafana Alerting]({{< relref "../../../alerting/notifications.md" >}}) and custom authentication to name a few.
+
+## Use cases for implementing a backend plugin
+
+The following examples should give you an idea of why you would consider implementing a backend plugin:
+
+- Enable [Grafana Alerting]({{< relref "../../../alerting/rules.md" >}}) for data sources.
+- Connect to non-HTTP services that normally cannot be connected to from a web browser, e.g. SQL database servers.
+- Keep state between users, e.g. query caching for data sources.
+- Use custom authentication methods and/or authorization checks not supported in Grafana.
+- Custom data source request proxy, see [Resources]({{< relref "#resources" >}}).
+
+## Grafana’s backend plugin system
+
+The Grafana backend plugin system is based on the [go-plugin library by HashiCorp](https://github.com/hashicorp/go-plugin). Grafana server launches each backend plugin as a subprocess and communicates with it over [gRPC](https://grpc.io/). This approach has a number of benefits:
+- Plugins can’t crash your grafana process: a panic in a plugin doesn’t panic the server.
+- Plugins are easy to develop: just write a Go application and go build (or use any other language which supports gRPC).
```

Comment:
Not sure what you mean by "and go build" here?

Reply:
Hah. It's same information as in current docs: https://grafana.com/docs/grafana/latest/developers/plugins/backend/#grafana-s-backend-plugin-system

But missed to denote `go build`:

```suggestion
- Plugins are easy to develop: just write a Go application and run `go build` (or use any other language which supports gRPC).
```

This is definitely a section we need to rewrite later.